modules/appeals_api/spec/support/shared_examples_for_icn_parameter_validation.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
```

Comment:
I added these shared examples to make the test cases for the ICN parameter behavior reusable, since we'll need the same tests for Appeals Status, Legacy Appeals, and Appealable Issues. This PR includes updates to tests for both Appeals Status and Legacy Appeals to use these shared examples - Appealable Issues will come later.